issue.go

```diff
@@ -1067,6 +1093,29 @@ func (s *IssueService) GetCustomFields(issueID string) (CustomFields, *Response,
 	return cf, resp, nil
 }
 
+// GetCustomFieldValues gets a list of the allowed values for a specifc customfield.
+func (s *IssueService) GetCustomFieldValues(project, issueTypeName, customField string) (*CustomFieldValue, error) {
+	apiEndpoint := fmt.Sprintf("/rest/api/2/issue/createmeta?projectKeys=%s&issuetypeNames=%s&expand=projects.issuetypes.fields", project, issueTypeName)
+	req, err := s.client.NewRequest("GET", apiEndpoint, nil)
+	if err != nil {
+		return nil, err
+	}
+	data := new(CreateMetaResponse)
+	resp, err := s.client.Do(req, data)
+	if err != nil {
+		jerr := NewJiraError(resp, err)
+		return nil, jerr
+	}
+	customFieldData, err := json.Marshal(data.Projects[0].IssueTypes[0].Fields[customField])
```

Comment:
it's possible to get out of range exceptions here. Can you add some additional validation of the response to ensure there's exactly 1 project, and exactly 1 issue type, and that the field exists?